index.js

```diff
@@ -114,7 +114,13 @@ module.exports = function create (opts) {
         menubar.window.setPosition(x, y)
       }
 
-      if (!opts['always-on-top']) menubar.window.on('blur', hideWindow)
+      if (!opts['always-on-top']) {
+        menubar.window.on('blur', hideWindow)
+      }
+
+      if (opts['show-on-all-workspaces'] !== false) {
```

Comment:
IMO we should use camelcase for all options, it is used everywhere else. @maxogden agree?
All options should also be added to the readme. :)